shell/platform/embedder/tests/embedder_config_builder.cc

```diff
@@ -35,8 +35,13 @@ EmbedderConfigBuilder::EmbedderConfigBuilder(
   opengl_renderer_config_.present = [](void* context) -> bool {
     return reinterpret_cast<EmbedderTestContext*>(context)->GLPresent();
   };
-  opengl_renderer_config_.fbo_callback = [](void* context) -> uint32_t {
-    return reinterpret_cast<EmbedderTestContext*>(context)->GLGetFramebuffer();
+  opengl_renderer_config_.fbo_with_frame_info_callback =
+      [](void* context, FlutterFrameInfo* frame_info) -> uint32_t {
+    FlutterFrameInfo tmp;
```

Comment:
Unclear why you prefer this form of copying of the struct instead of just `return reinterpret_cast<EmbedderTestContext*>(context)->GLGetFramebuffer(*frame_info)`. That is more resilient to updates to the frame info struct.

Reply:
copy-pasta error.